src/components/List/index.tsx

```diff
@@ -130,6 +159,10 @@ const useStyles = makeStyles((theme: Theme) =>
         borderRadius: '20px',
       },
     },
+    listMui: {
+      marginLeft: '20px',
+      borderLeft: '1px solid #e1e1e1',
```

Comment:
Also here, can we use the closest color from our theme?

Reply:
Why are we mixing styled components with classes?

Reply:
I replaced it with `gray500`